app/controllers/case_logs_controller.rb

```diff
@@ -20,22 +20,19 @@ def edit
     render :edit
   end
 
-  def next_page
+  def submit_form
```

Comment:
Is this submitting the form or technically just the question(s) you've answered on the current page?